doc/README.md

```diff
@@ -0,0 +1,2 @@
+Information: This directory is not a documentation source any more. Developer docs are contained in 
```

Comment:
No problem with that. It contains an old structure for the user docs. If you consider this outdated just drop the "doc" dir.